src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,33 @@
+.page__catalog {
```

Comment:
`page__catalog` must be in page.scss file

Reply:
Fixed (add page.scss)